docs/Coiote_IoT_DM/Visualize_data.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Data Visualization
+date: 2023-05-02
+og_title: AVSystem IoT Developer Zone
+---
+
+# Visualize device data
+
+The **Dashboard** consists of **widgets** which display telemetry data. In order to collect data to display, **observations** needs to be set on specific **resources**. 
```

Comment:
This is not entirely true. Right now, Coiote collects the data that is later displayed on dashboard from multiple sources:
- from NOTIFY messages, when you set an observation on a given resource,
- from SEND messages, which happen once device decides to send such message for a given resource; Coiote cannot directly trigger such SEND,
- from READs executed on device, for example during refresh datamodel

So I guess I would reword it to 
```
In order to collect sample data to display, one of following needs to happen:
- **observation** is set on a resource and device sends notifications with the resource value,
- **read** is invoked on either a resource or entire device data model,
- device itself sends a LwM2M SEND message with the updated resource value.
```

Reply:
Thanks! Updated.